### ISSUE TITLE
CloudCompare Cannot Acess Mounted Hard Disk Drive

### ISSUE THREAD

**Issue by sunstarchan** [CloudCompare/CloudCompare#1257]
I'm working on Ubuntu System. There are two disk drives, and one is for system the other one for data storage. I put some PLY files in the other drive mounted to the system other than computer. When I open a PLY files, it says "reading error (no access right?)". But it works well when I copy the PLY file to my home directory. 

**Your environment**
 - CC Version (or git hash/tag): 2.10.1
 - OS & Version: Ubuntu 18.04.1
 - Graphics card: NVIDIA GTX 2060
 - Qt Version (if compiling): 5.7.22

**Comment by PE1NUT** [comment]
This is a known problem when using the Snap package, see issue 778.
If you are using the snap, you could try removing it, and using the Ubuntu package instead.

**Comment by sunstarchan** [comment]
@asmaloney  Yes, I installed cloudcompare by using the snap, and I followed the link in #778 to install cloudcompare with the deb package. As I'm using Ubuntu 18.04, the dependency libswscale5 is from Ubuntu distribution groovy, so cloudcompare cannot be installed correctly.

**Comment by rjanvier** [comment]
Hello, it's a snap sandboxing "limitation". You should grant permission to the package via the software center or via `snap connect cloudcompare:removable-media`, I never managed to reproduce the bug int #778 (the removable-media plug that disconnect after each use)

**Comment by kaitaotang** [comment]
sudo snap refresh -- stable
snap install cloudcompare
sudo snap refresh
sudo snap connect  cloudcompare:removable-media  :removable-media

**Comment by kaitaotang** [comment]
I fixed this problem by using these commands:
sudo snap refresh -- stable
snap install cloudcompare
sudo snap refresh
sudo snap connect  cloudcompare:removable-media  :removable-media
